app/src/main/java/com/example/kharrison/my_application/CalcFunctions.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.kharrison.my_application
+
+
+class CalcFunctions {
+
+   fun add(leftNum : Int, rightNum  : Int) = leftNum + rightNum
+   fun subtract(leftNum : Int, rightNum : Int) = leftNum - rightNum
+   fun multiply(leftNum : Int, rightNum : Int) = leftNum * rightNum
+   fun divide(leftNum : Int, rightNum : Int) = leftNum / rightNum
```

Comment:
Kotlin does have named functions for the basic operators, so you *might* be able to just use method references to those:  `Int.plus()`, `Int.minus()`, `Int.times()` and `Int.divi()`